dspace-api/src/main/java/org/dspace/content/service/ItemService.java

```diff
@@ -539,11 +539,11 @@ public void move(Context context, Item item, Collection from, Collection to, boo
         throws SQLException, AuthorizeException, IOException;
 
     /**
-     * Check the bundle ORIGINAL to see if there are any uploaded files
+     * Check the bundle ORIGINAL and METADATA to see if there are any uploaded files
```

Comment:
@milanmajchrak this is dpace-core? Anything to do with CMDI will never get to upstream

Reply:
Changed